docs/source/docker/security/security.rst

```diff
@@ -0,0 +1,83 @@
+Security
+==========
+
+Security and data privacy is very important to us. In this section
+you find all security related docs. 
+Legal documents such as Privay Notice, T&C, DPA are available under
+`lightly.ai/legal <https://lightly.ai/legal>`_.
+
+.. note:: These docs apply to all personal, team and custom account plans.
+
+Architecture Overview
+----------------------
+
+Here you find an overview of the cloud architecture. 
+
+**A few important things to note:**
+
+- Data storage and processing happens on clients infrastructure 
+- Lightly only needs permission to list the cloud bucket and to create signed URLs
```

Comment:
to list the cloud bucket -> to list files within your cloud storage (S3, GCS, Azure)